certbot-ci/windows_installer_integration_tests/conftest.py

```diff
@@ -22,3 +24,26 @@ def pytest_addoption(parser):
                                           'nsis', 'certbot-beta-installer-win32.exe'),
                      help='set the path of the windows installer to use, default to '
                           'CERTBOT_ROOT_PATH\\windows-installer\\build\\nsis\\certbot-beta-installer-win32.exe')
+    parser.addoption('--confirm', action='store_true',
+                     help='if set, this test will not ask for user confirmation before running')
+
+
+def pytest_configure(config):
+    """
+    Standard pytest hook used to add a configuration logic for each node of a pytest run.
+    :param config: the current pytest configuration
+    """
+    if not config.option.confirm:
+        capture_manager = config.pluginmanager.getplugin('capturemanager')
+        try:
+            capture_manager.suspendcapture(in_=True)
```

Comment:
It looks like these [functions have been renamed in newer versions of pytest](https://stackoverflow.com/questions/50638571/in-pytest-how-to-config-capturemanager-plugin-error-capturemanager-object-has) and I'm not sure pytest considers them part of their stable API as they made the change after pytest 3.0 but before pytest 4.0.

Unless there's a more stable way to do this, how about we just raise an assertion saying that the flag must be set to run these tests?

Reply:
Ok. Implemented.